starknet_py/net/client.py

```diff
@@ -197,6 +197,24 @@ async def estimate_fee(
         :return: Estimated amount of Wei executing specified transaction will cost
         """
 
+    @abstractmethod
+    async def estimate_fee_bulk(
+        self,
+        txs: List[Union[InvokeFunction, Declare, DeployAccount]],
```

Comment:
I'm not sure if we should name it `txs` or `transactions` :hmm: @cptartur ?

Reply:
Maybe `transactions`? I usually prefer more explicit names.